doc/api/packages.md

```diff
@@ -704,18 +704,17 @@ is provided below to assist with ecosystem coordination.
 
 * `"types"` - can be used by typing systems to resolve the typing file for
   the given export. _This condition should always be included first._
-* `"deno"` - indicates a variation for the Deno platform.
 * `"browser"` - any web browser environment.
-* `"react-native"` - will be matched by the React Native framework (all
-  platforms). _To target React Native for Web, `"browser"` should be specified
-  before this condition._
 * `"development"` - can be used to define a development-only environment
   entry point, for example to provide additional debugging context such as
   better error messages when running in a development mode. _Must always be
   mutually exclusive with `"production"`._
 * `"production"` - can be used to define a production environment entry
   point. _Must always be mutually exclusive with `"development"`._
 
+For more conditions that relate to runtimes like `"deno"` or `"bun"`, see the
+[Runtime Keys][] Draft Community Group Report by [WinterCG][].
```

Comment:
I think it would help to give the context specifically that these are platform keys:

```suggestion
For other runtimes, platform-specific condition key definitions are maintained by the
[WinterCG][] in the [Runtime Keys][] specification.
```